frontend/src/components/mspcontracts/BeneficiaryContractPeriod.vue

```diff
@@ -30,7 +30,17 @@ export default {
       return decodeRelationship(this.contractPeriod.relationship)
     },
     cancelReason() {
-      return this.contractPeriod.cancelReason === 'E' ? 'Eligible' : ''
+      const cancelReason = ''
```

Comment:
There's not too much point to declaring this a const and then only using it once in the default statement. I would just return the empty string directly.